lib/telegraph/client.rb

```diff
@@ -0,0 +1,67 @@
+module Telegraph
+  class Client
+    include Telegraph::Connection
+
+    attr_reader :token
+
+    def initialize(token)
+      @token = token
+    end
+
+    def create_account(attrs = {})
+      acc_info = get('createAccount', attrs)
+      @account = Account.new(acc_info)
```

Comment:
So, one account per client? Why?

Reply:
(Reading a bit more through API docs)
Well, it seems that there are one account object per **token**, while "client" can manage several accounts (with different tokens), so probably the entire structure should be thought about a bit more?..